### PR TITLE
*: bump etcd verison to 3.2.13

### LINKED ISSUE #1813: etcd version: bump to 3.2.13
There is a bug when shutting down etcd: https://github.com/coreos/etcd/issues/8916

### PR DESCRIPTION
**Pull request by fanminshi**
bump etcd version to 3.2.13 in many places.
exposed DefaultVersion allowing backup_restore_test.go to use it to compare 
backup status with the default etcd version.

fixes #1813

### CODE REVIEW

**Comment by fanminshi** [comment]
cc/ @hongchaodeng @hasbro17

**Comment by hongchaodeng** [comment]
Please pay attention to:
- there are still many places containing "3.2.11"
- CHANGELOG

**Comment by fanminshi** [comment]
@hongchaodeng tried again. PTAL

File: pkg/apis/etcd/v1beta2/cluster.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 
 const (
 	defaultRepository = "quay.io/coreos/etcd"
-	defaultVersion    = "3.2.11"
+	DefaultVersion    = "3.2.13"
```
Comment (hongchaodeng) [inline pkg/apis/etcd/v1beta2/cluster.go]: DefaultEtcdVersion

Comment (fanminshi) [reply pkg/apis/etcd/v1beta2/cluster.go]: fixed!

**Comment by fanminshi** [comment]
merge when green

**Comment by hongchaodeng** [comment]
Before merging, please check the e2eslow log to verify the panic message is gone.

**Comment by fanminshi** [comment]
will do!

**Comment by fanminshi** [comment]
no panic is the e2eslow logs. merging.

**Pull request merged**